server/backend/readme.md

```diff
@@ -128,7 +136,7 @@ Assuming you've already installed the dependencies from requirements.txt, you ju
 
 ```
 cd server/backend
-pip install -r requirements-dev.txt
+pip3 install -r requirements-dev.txt
```

Comment:
I think it's okay to leave pip install, if the user only installed python3, pip and pip3 will do the same I believe

Reply:
Thanks, I changed it back, although I thought they are probably the same, I am not entirely clear how python/python3/pip/pip3 work in Python world, confusing.